### DISCUSSION TITLE
sqlite data type have no DECIMAL

### DISCUSSION THREAD

**Discussion by anlexN** [prisma/prisma#13613] [Q&A]
![image](https://user-images.githubusercontent.com/8686739/171540048-275d6a9c-cb22-4c84-9a79-1e329a7b79cd.png)
**Official docs:** [Datatypes In SQLite](https://www.sqlite.org/datatype3.html)

**Comment by austincrim** [comment]
Hey @anlexN,

Could you clarify your question a bit more? I'm not sure what you are asking.

**Reply by janpio** [reply]
`Decimal` used to be mapped to `REAL` in our SQLite implementation, but as `Decimal` should be a proper decimal type, we needed to change it to `DECIMAL` which has an affinity to `NUMERIC` which is more like that.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
